.github/workflows/build-master.yaml

```diff
@@ -27,21 +27,14 @@ jobs:
           java-version: 17
           distribution: temurin
           cache: gradle
-      - uses: actions/cache@v3
```

Comment:
ingen cahe?

Reply:
Den er på L29 :)

Reply:
Yes, cache: i setup-java bruker actions/cache, så det blir att man drar det in dubbelt her.